flow/alerting/slack_alert_sender.go

```diff
@@ -7,34 +7,46 @@ import (
 	"github.com/slack-go/slack"
 )
 
-type slackAlertSender struct {
+type SlackAlertSender interface {
+	AlertSender
+}
+
+type slackAlertSenderImpl struct {
```

Comment:
```suggestion
type SlackAlertSender struct {
```
Don't need that much abstraction